Sources/SKTestSupport/SwiftPMTestWorkspace.swift

```diff
@@ -0,0 +1,70 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+import LanguageServerProtocol
+import SKCore
+import TSCBasic
+
+public class SwiftPMTestWorkspace: MultiFileTestWorkspace {
```

Comment:
Does this end up being significantly slower than the single file case? If not we could just use SwiftPMTestWorkspace for all the things (with a simpler constructor for the single file case).

Reply:
Unfortunately, yes. It’s about 1.3s vs 300ms, IIRC (measured yesterday). IMO it’s OK to pay that price for tests that actually need cross-file functionality and so we don’t have to use the complexity of TIBS but I don’t want to pay it for every test case.